Adyen/Components/Base/PaymentComponentData.swift

```diff
@@ -78,10 +84,12 @@ public struct PaymentComponentData: ShopperInformation {
     ///   - paymentMethodDetails: The payment method details submitted from the payment component.
     ///   - storePaymentMethod: Whether the user has chosen to store the payment method.
     ///   - browserInfo: The device default browser info.
-    public init(paymentMethodDetails: PaymentMethodDetails, storePaymentMethod: Bool = false, browserInfo: BrowserInfo? = nil) {
+    ///   - addressInfo: The payment's billing address.
+    public init(paymentMethodDetails: PaymentMethodDetails, storePaymentMethod: Bool = false, browserInfo: BrowserInfo? = nil, billingAddress: AddressInfo? = nil) {
```

Comment:
instead of passing the `billingAddress` its should be in the `PaymentMethodDetails` like this:
```
public var billingAddress: AddressInfo? {
        guard let shopperInfo = paymentMethod as? ShopperInformation else { return nil }
        return shopperInfo. billingAddress
    }
```

Reply:
Good one!